docs/user/callbacks.rst

```diff
@@ -48,44 +48,51 @@ on_train_begin
 ^^^^^^^^^^^^^^
 
 Called once at the start of the training process (e.g. when calling
-fit).
+fit).  Signature: `on_train_begin(net, X, y)`.
```

Comment:
Maybe paste signatures directly into the headers? There are precedents for doing this already: https://skorch.readthedocs.io/en/latest/user/neuralnet.html#fit-x-y